csrankings-9.csv

```diff
@@ -1291,7 +1291,7 @@ Yan Tong,University of South Carolina,http://www.cse.sc.edu/~tongy,tY9bQTkAAAAJ
 Yan Wang 0002,Macquarie University,http://web.science.mq.edu.au/~yanwang,3h6_oVEAAAAJ
 Yan Wang 0003,Temple University,https://www.binghamton.edu/cs/people/ywang.html,NOSCHOLARPAGE
 Yan Xiong,USTC,http://dsxt.ustc.edu.cn/zj_ywjs.asp?zzid=709,X08c_CQAAAAJ
-Yan Yan 0002,Texas State University,http://cs.txstate.edu/~y_y34,zhi-j1wAAAAJ
+Yan Yan 0002,Illinois Institute of Technology,http://cs.txstate.edu/~y_y34,zhi-j1wAAAAJ

```

Comment:
Home page still indicates Texas State affiliation. Please point to an official site that indicates a change of affiliation.